features/gezag-minderjarige/1.3-is-naar-buitenland-geemigreerd-geweest.feature

```diff
@@ -0,0 +1,94 @@
+  
+  Gebruikte velden:
+    - Geboorteland -> 01.03.30
+    - Datum vestiging in Nederland -> 08.14.20
+
+    Achtergrond:
+      Gegeven de persoon 'Ruth' met burgerservicenummer '000000012'
+      * is meerderjarig
+      En de persoon 'Maurice' met burgerservicenummer '000000024'
+      * is meerderjarig
+      En 'Ruth' en 'Maurice' zijn met elkaar gehuwd
+      En de persoon 'Lisa' met burgerservicenummer '000000036'
+      * is minderjarig
+      * is in Nederland geboren
+      * is ingeschreven in de BRP
+      * heeft 'Ruth' als ouder 1
+      * heeft 'Maurice' als ouder 2
+
+  Regel: Kind is nooit geëmigreerd geweest
+
+    Scenario: Het kind is geboren in Nederland en verblijft in Nederland er is sprake van TweehoofdigOuderlijkGezag
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde                    |
+      | type                             | TweehoofdigOuderlijkGezag |
+      | minderjarige.burgerservicenummer | 000000036                 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000012 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000024 |
+      
+  Regel: Het kind is naar het buitenland geëmigreerd
+
+    Scenario: Het kind is geboren in Nederland en is geëmigreerd er is sprake van TweehoofdigOuderlijkGezag
+      Gegeven persoon 'Lisa'
+      * is geëmigreerd naar het buitenland
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde                    |
+      | type                             | TweehoofdigOuderlijkGezag |
+      | minderjarige.burgerservicenummer | 000000036                 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000012 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000024 |
+      
+  Regel: Het kind is naar het buitenland geëmigreerd geweest
+
+    Scenario: Het kind is geboren in Nederland, is geëmigreerd geweest en daarna terug naar Nederland gekomen er is sprake van TweehoofdigOuderlijkGezag
```

Comment:
geboren in NL zou ik als aparte rule met scenario's illustreren. Zie bovenstaand voorstel.

Reply:
Kan je dit verder toelichten? Voor zover ik het begrijp zijn

> Het kind is naar het buitenland geëmigreerd **geweest**

en

> Het kind is geboren in Nederland, is geëmigreerd ~~geweest~~ en daarna terug naar Nederland gekomen

inhoudelijk dezelfde zinnen. De tweede zin gebruikt alleen meer worden.

Reply:
@FrozenSync deze 2 zinnen betekenen niet hetzelfde, want iemand die niet in Nederland geboren is kan ook geëmigreerd geweest zijn. Alleen is die remigratie lastig te onderscheiden in de BRP.

Reply:
Ik begrijp het niet. Als je geëmigreerd bent geweest maakt het toch niet meer uit of je wel of niet in NL bent geboren? In beide gevallen wordt immers geen gezag bepaald. Tenzij er een uitspraak of een adoptie is. Maar dan maakt het ook niet meer uit of je in NL bent geboren of niet. Dan is de uitspraak/adoptie leidend.

Reply:
> Of zeg je dat als je niet in NL bent geboren, en je bent geremigreerd geweest, dat je daar dan niet op kunt afvallen (en je dus geboorteland nodig hebt)...

Dit is precies wat er gebeurd in de implementatie. De remigratie vraagstuk wordt alleen toegepast op kinderen die in Nederland zijn geboren. Kinderen die in het buitenland zijn geboren slaan het over. 

Daaruit volgt dat de volgende aanname

> Als je geëmigreerd bent geweest maakt het toch niet meer uit of je wel of niet in NL bent geboren?

niet klopt.

De waarom weet ik niet. Wellicht kan Frank daar meer over vertellen. Het lijkt erop dat het achterhalen van remigratie voor kinderen die in het buitenland zijn geboren moeilijk is.

Reply:
@fsamwel zou je hierop kunnen reageren?

Reply:
bovenstaande zet ik om naar een apart issue in ready maken en resolve ik de comment hier.

https://github.com/BRP-API/brp-api-gezag/issues/222